release/models/bfd/openconfig-bfd.yang

```diff
@@ -270,6 +277,14 @@ module openconfig-bfd {
         "When this leaf is set to true - BFD will be enabled on
         each member interface of the aggregated Ethernet bundle.";
     }
+
+    leaf enable-echo {
+      type boolean;
+      default false;
```

Comment:
The RFC doesn't mention any default value for echo mode.  (I didn't see it here https://datatracker.ietf.org/doc/html/rfc5880#section-6.8.13).  Please remove the default as OC prefers to not define defaults.